test/object-store/sync/flx_sync.cpp

```diff
@@ -3299,6 +3300,79 @@ TEST_CASE("flx: bootstrap changesets are applied continuously", "[sync][flx][app
     CHECK(user_commit_version == bootstrap_version + 1);
 }
 
+TEST_CASE("flx: async open + register subscription callack while bootstrapping", "[sync][flx][async-open]") {
```

Comment:
You should load some initial data and check at the end that the data is received:
```
auto foo_obj_id = ObjectId::gen();
    harness.load_initial_data([&](SharedRealm realm) {
        CppContext c(realm);
        Object::create(c, realm, "TopLevel",
                       std::any(AnyDict{{"_id", foo_obj_id},
                                        {"queryable_str_field", "foo"s},
                                        {"queryable_int_field", static_cast<int64_t>(5)},
                                        {"non_queryable_field", "created as initial data seed"s}}));
    });
...
// Check foo_obj_id exists in the realm
....
```

Reply:
should be fine now.